src/e2e/java/teammates/e2e/util/LNPTestData.java

```diff
@@ -59,6 +59,7 @@ public DataBundle generateJsonData() {
 
     /**
      * Returns list of header fields for the data in the CSV file to be generated.
+     * Note that these headers should correspond to the variables used in the JMeter L&P test.
```

Comment:
This is already slightly more implementation specific. In Javadoc, the usual practice is the first line tells briefly what the method does, but nothing more. If you need something specific like this, the subsequent paragraphs are the place for it.
```java
/**
 * Returns list of header fields for the data in the CSV file to be generated.
 *
 * <p>Note that these headers should correspond to the variables used in the JMeter L&P test.
 */
```